kevm-pyk/src/tests/integration/test_integration.py

```diff
@@ -1,5 +1,77 @@
+import json
+
+from pyk.kast.inner import KSort
+
 from kevm_pyk.hello import hello
+from kevm_pyk.solc_to_k import Contract
 
 
 def test_hello() -> None:
     assert hello('World') == 'Hello, World!'
+
+
+contract_json = """
+{
+  "abi": [
+    {
+      "inputs": [
+        {
+          "components": [
+            {
+              "internalType": "address",
+              "name": "target",
+              "type": "address"
+            },
+            {
+              "internalType": "bytes",
+              "name": "callData",
+              "type": "bytes"
+            }
+          ],
+          "internalType": "struct IMulticall3.Call[]",
+          "name": "calls",
```

Comment:
These seem like they can go as unit tests, not integration tests, because they don't require any external programs, they're just testing functionality of the python code.

Reply:
Thanks, I wasn't sure how we defined integration tests in the context of this project. I've moved it over to the unit tests file.